src/adapters/scene_direction.js

```diff
@@ -18,6 +20,10 @@ export default class SceneDirection {
     this.directionPanel = window.app.directionPanel;
     this.mapFeaturesByRoute = {};
 
+    const iconsBaseUrl = nconf.get().system.baseUrl + 'statics/images/direction_icons/';
+    this.addMapImage(`${iconsBaseUrl}/walking_bullet_active.png`, 'walking_bullet_active');
+    this.addMapImage(`${iconsBaseUrl}/walking_bullet_inactive.png`, 'walking_bullet_inactive');
```

Comment:
Very minor: this introduces a double slash `direction_icons//...` in the icons URLs.

Reply:
Thanks, I fixed it!